src/User_Command.php

```diff
@@ -162,7 +162,17 @@ public function list_( $args, $assoc_args ) {
 
 		$assoc_args['count_total'] = false;
 		$assoc_args                = self::process_csv_arguments_to_arrays( $assoc_args );
-		$users                     = get_users( $assoc_args );
+
+		if ( ! empty( $assoc_args['role'] ) && 'none' === $assoc_args['role'] ) {
```

Comment:
@wp-cli/committers Do you think this should be `--role=none` or `--no-role`? I think I'm leaning towards the latter...

Reply:
I was following this https://github.com/wp-cli/entity-command/issues/34#issuecomment-317870209

We can change it to whatever you say. Just let me know. :)

When you say `--no-role` means the command should be wp user list `--no-role` ?

Reply:
> When you say `--no-role` means the command should be `wp user list --no-role` ?

Yep, exactly.

> We can change it to whatever you say. Just let me know. :)

@johnbillion @schlessera Do either of you have an opinion?

Reply:
I think `--no-role` might be misleading, as this is typically used for boolean flags, and would make me assume `--role` (without any value) is a thing, too.
I'd prefer `--role=none` here to make it clear that this is not a boolean flag.

Reply:
+1 for `--role=none` which matches the URL used in wp-admin: `?role=none`.

Reply:
@johnbillion is right.

Thank You all guys. So we'll keep `--role=none`

I'll start adding behat test now

Reply:
I think we can Resolve this conversation. Let me know.

Thanks